### ISSUE TITLE
[DoomTools] Updater is Currently Broken (Connection Reset)

### ISSUE THREAD

**Issue by MTrop** [MTrop/DoomTools#109]
Contacting Github API via the updater (`--update`) yields this:

```
Current version is: XXXXXXXXXXXXXXXX
Contacting update site...
ERROR: Read error contacting update site: Connection reset
```

Maybe caused by agent? Maybe caused by other reason on the HTTPS connection?

**Comment by biwa** [comment]
I can confirm the issue, I'm getting same same message. Interestinly accessing both `https://api.github.com` and `https://api.github.com/repos/MTrop/DoomTools/releases/latest` works  fine with both PowerShell's `Invoke-WebRequest` and using Python `requests`. With the latter it even works if I use a nonsensical `User-Agent` header.

If possible you could check the actual HTTP status code.

**Comment by MTrop** [comment]
I don't even receive a status code. I get a RESET packet back from the server. This makes me think it could be a protocol issue and GitHub updated something on their end, perhaps?

Hmmm... now I'm getting timeouts. Maybe this is an intermittent thing?

**Comment by MTrop** [comment]
It looks like I'm missing some headers that GitHub is now strict about. Need to experiment further...

**Comment by MTrop** [comment]
Fixed in commit 2b90a20b00c364a6030fcb9997f6aa45d46cb261.

I was missing `Accept` headers. Also, I'm now sending a user agent string.
